### PR TITLE
mpv: update to 0.33.0

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

mpv: update to 0.33.0

- Drop libsmbclient, because upstream dropped. (why?)
- Disable build date

Package(s) Affected
-------------------

No

Security Update?
----------------

No

Build order
----------------

libarchive -> mpv

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by MingcongBai** [comment]
@eatradish No need to mark "DO NOT MERGE" in title. Convert this pull request to a "draft" and note your reasoning below.

File: extra-multimedia/mpv/autobuild/build [hunk 1]
```diff
@@ -23,14 +24,12 @@ install -m644 "$SRCDIR"/etc/mpv.conf "$PKGDIR"/etc/mpv/mpv.conf
...
+cd "$SRCDIR"/mpv-bash-completion
+make ${MAKEFLAGS}
 
-#cd "$SRCDIR"/mpv-bash-completion-${COMPLETION_GEN_VER}
-#make ${MAKEFLAGS}
-
-#mkdir -p "$PKGDIR"/etc/bash_completion.d
-#cp -av mpv "$PKGDIR"/etc/bash_completion.d/
-#cd "$SRCDIR"
+mkdir -p "$PKGDIR"/etc/bash_completion.d
```
Comment (MingcongBai) [inline extra-multimedia/mpv/autobuild/build]: Verbosity here.

File: extra-multimedia/mpv/autobuild/build [hunk 2]
```diff
@@ -23,14 +24,12 @@ install -m644 "$SRCDIR"/etc/mpv.conf "$PKGDIR"/etc/mpv/mpv.conf
...
+cp -av mpv "$PKGDIR"/etc/bash_completion.d/
```
Comment (MingcongBai) [inline extra-multimedia/mpv/autobuild/build]: Absolute path here please.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-multimedia/mpv/autobuild/build [hunk 3]
```diff
@@ -7,30 +7,29 @@ abinfo "Configuring..."
                 --enable-dvbin \
                 --enable-openal \
                 --enable-sdl2 \
-                --enable-libsmbclient
+                --enable-libarchive \
+                --disable-build-date
 
 abinfo "Building..."
 ./waf build
```
Comment (MingcongBai) [inline extra-multimedia/mpv/autobuild/build]: More absolute path here.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Comment by OriginCode** [comment]
Not LGTM, ERR: 
`mpv: symbol lookup error: mpv: undefined symbol: archive_read_support_format_rar5`

image

**Comment by eatradish** [comment]
> Not LGTM, ERR:
> `mpv: symbol lookup error: mpv: undefined symbol: archive_read_support_format_rar5`
> 
> image

@OriginCode How about trying again after upgrading with atm?

**Comment by eatradish** [comment]
image

LGTM on AMD64

**Comment by MingcongBai** [comment]
LGTM on POWER.

图片

**Comment by OriginCode** [comment]
Works but with err (amd64):
image

**Comment by MingcongBai** [comment]
> Works but with err (amd64):
> image

This is probably an NVIDIA issue.

**Pull request merged**